zebrad/tests/acceptance.rs

```diff
@@ -264,11 +264,11 @@ fn help_no_args() -> Result<()> {
         is_zebrad_version,
         &output.output.stdout,
         "stdout",
-        "a valid zebrad semantic version",
+        "with a valid zebrad semantic version",
```

Comment:
Nitpick: This change isn't quite grammatically correct:
https://github.com/ZcashFoundation/zebra/blob/618d3fcca0f27a38ef0d7ce1c1d8e0436cc0117b/zebra-test/src/command.rs#L1129

```suggestion
        "are valid zebrad semantic versions",
```